### PR TITLE
added instructions to the resources page

### PR DESCRIPTION
**Pull request by daniellehu**
Added link to the tutorial PDF in the resources page. 

Looks like this:
image

### CODE REVIEW

File: resources.html [hunk 1]
```diff
@@ -57,7 +57,7 @@
 							<a class="page-scroll" href="./index.html">Home</a>
 						</li>
                         <li>
-                            <a class="page-scroll" href="http://54.191.129.39/">Get Started</a>
+                            <a class="page-scroll" href="http://54.191.129.39/" target="_blank">Get Started</a>
```
Comment (blengerich) [inline resources.html]: hardcoded address

Comment (daniellehu) [reply resources.html]: I think it was hardcoded before?

Comment (blengerich) [reply resources.html]: yeah, you're right it was. since we're messing with the line anyway, might as well fix it.

Comment (daniellehu) [reply resources.html]: @blengerich can you take a look at the new change? I just changed the IP address reference to "http://app.genamap.org/". But I wasn't sure if you meant writing a script that defined variables for each link and using those variables in place.

Comment (blengerich) [reply resources.html]: Looks good, thanks.

**Pull request merged**